gms/feature_service.cc

```diff
@@ -193,7 +194,7 @@ void feature::enable() {
 
 db::schema_features feature_service::cluster_schema_features() const {
     db::schema_features f;
-    f.set_if<db::schema_feature::VIEW_VIRTUAL_COLUMNS>(view_virtual_columns);
+    f.set<db::schema_feature::VIEW_VIRTUAL_COLUMNS>();
```

Comment:
At one point we can also grandfather the schema feature too.

Reply:
(same for the other ones below)

Reply:
Maybe I should do it here.

Reply:
done